src/templates/website/index.html

```diff
@@ -0,0 +1,88 @@
+{% from "_macros/app_tile.html" import app_tile %}
```

Comment:
you could probably extra some things out into a macro and share them between the app detail and website detail